fabric/bin/lib/common_ledger.sh

```diff
@@ -131,9 +148,14 @@ ledger_init() {
     try ${CONFIGTXGEN_CMD} -channelID ${CHAN_ID} -profile SampleSingleMSPChannel -outputCreateChannelTx ${CHANNEL_TX}
     # - create genesis block, only by one peer
     try ${PEER_CMD} channel create -o ${ORDERER_ADDR} -c ${CHAN_ID} -f ${CHANNEL_TX} --outputBlock ${CHANNEL_BLOCK}
+    para
+    echo "[+] Channel has been created successful...."
+    echo "[-] JOINING CHANNEL FOR NOW IS DISABLED."
```

Comment:
Could you please elaborate why we don't join the channel?

Reply:
ledger_init doesn't work yet as ercc would need the decorator yet we disabled the corresponding build. Hence the decorator definition in core.yaml is commented out and here related functionality is commented out.  It probably would be nicer, thoyugh, to only disable the ercc transactions but still install it and also join the channel  ..